packages/headless/src/modules/user-auth/user-auth.module.ts

```diff
@@ -0,0 +1,38 @@
+import { Module } from '@nestjs/common';
+import { ConfigModule, ConfigService } from '@nestjs/config';
+import { JwtModule } from '@nestjs/jwt';
+import { PassportModule } from '@nestjs/passport';
+import { ResolveDatabaseDependency } from 'src/database/database.resolver';
+import { HelperService } from 'src/helper/helper.service';
+import { UserAuthController } from './controllers/auth.controller';
+import { UserProfileController } from './controllers/profile.controller';
+import { JwtStrategy } from './passport/jwt.strategy';
+import { UserAuthRepository } from './repository';
+import { IUserAuthDB } from './repository/user.db.interface';
+import { UserAuthService } from './services/user-auth.service';
+import { UserProfileService } from './services/user-profile.service';
+
+@Module({
+  imports: [
+    PassportModule,
+    JwtModule.registerAsync({
```

Comment:
Can we move it somewhere and import here rather than writing all the logic in import